File: week1/numberguessing.py
from random import randint

number_guessed = int(input("Guess from 1 to 10: "))
number_chosen = randint(1,10)
print(number_chosen)
if (number_guessed == number_chosen):
    print("Correct!")
elif (number_guessed > number_chosen):
    print ("Higher!")
else:
    print ("Lower")

File: week3/learning/javascript/numberguessing.js
let number_attempt = 0
let number_to_guess =  Math.floor((Math.random() * 10) + 1);
let number_guessed;
while (number_guessed != number_to_guess) {
    number_guessed = Number(prompt("Try to guess again: "))
    if (number_guessed > number_to_guess) console.log("Too high")
    else if (number_guessed < number_to_guess) console.log("Too low")
    number_attempt += 1
}
console.log(`Correct! You tried ${number_attempt} times`)    